### PR TITLE
Notes features

### PR DESCRIPTION
**Pull request by ravikumar1002**
Create notes  with  label and priority
 Update notes

### CODE REVIEW

File: src/components/note-card/NoteCard.jsx [hunk 1]
```diff
@@ -1,23 +1,80 @@
...
+        label: "",
+        color: "",
+        pin: false,
+    };
+
+    const [notesData, setNotesData] = useState(
+        updateNotes ? { ...notesValue } : { ...defaultValue }
+    );
+
+console.log(notesData)
```
Comment (vermasavita) [inline src/components/note-card/NoteCard.jsx]: Remove console if not needed

File: src/components/save-notes-card/SaveNotes.jsx [hunk 2]
```diff
@@ -1,37 +1,86 @@
...
+                            </p>
+                        </div>
+                    </div>
+                    <div className=" p-1 d-flex gap-1">
+                        {userCreatedNotes.priority && (
+                            <div className="btn-secondary btn-x-sm border-pill">
+                                {userCreatedNotes.priority}
+                            </div>
+                        )}
+                        {/* {userCreatedNotes.label.length > 0 &&
```
Comment (vermasavita) [inline src/components/save-notes-card/SaveNotes.jsx]: Remove comment if not needed.

**Review by vermasavita — APPROVED** [review]
Just remove the comments. Rest looks good to me.

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks clean with proper naming with good UI

Need to check ->

Label is not working 

image
image

**Comment by ravikumar1002** [comment]
> code looks clean with proper naming with good UI
> 
> Need to check ->
> 
> Label is not working
> 
> image image

Actually you need to press enter key to add label now I have changed UI now you can either click enter or hit add icon button

**Comment by chanchal-panpaliya** [comment]
> > code looks clean with proper naming with good UI
> > Need to check ->
> > Label is not working
> > image image
> 
> Actually you need to press enter key to add label now I have changed UI now you can either click enter or hit add icon button

My bad, it's working properly.

**Comment by ravikumar1002** [comment]
> 

thanks after that I got another bug that was important to fix

**Comment by ravikumar1002** [comment]
> > > code looks clean with proper naming with good UI
> > > Need to check ->
> > > Label is not working
> > > image image
> > 
> > 
> > Actually you need to press enter key to add label now I have changed UI now you can either click enter or hit add icon button
> 
> My bad, it's working properly.

thanks for reviewing,  after that I got another bug that was important to fix

**Pull request merged**